packages/xdg_directories/test/xdg_directories_test.dart

```diff
@@ -62,21 +62,21 @@ XDG_VIDEOS_DIR="$HOME/Videos"
 
   test('Default fallback values work', () {
     fakeEnv.clear();
-    fakeEnv['HOME'] = tmpDir.path;
-    expect(xdg.cacheHome.path, equals(testPath('.cache')));
-    expect(xdg.configHome.path, equals(testPath('.config')));
-    expect(xdg.dataHome.path, equals(testPath('.local/share')));
+    fakeEnv['HOME'] = tmpDir!.path;
+    expect(xdg.cacheHome!.path, equals(testPath('.cache')));
+    expect(xdg.configHome!.path, equals(testPath('.config')));
+    expect(xdg.dataHome!.path, equals(testPath('.local/share')));
```

Comment:
Any of these that can return null (see comments in the other file) should be expecting non-null results, not asserting them.